### DISCUSSION TITLE
Table numbers and cross-references with `parse-latex` extension in Quarto document

### DISCUSSION THREAD

**Discussion by friendly** [quarto-dev/quarto-cli#10432] [Q&A]
### Description

I found the `parse-latex` lua extension, https://github.com/tarleb/parse-latex and it looks good for making tables of math expressions in my Quarto book document. However, I can't figure out how to use it to generate a table with table number, caption and ability to xreference in a document.

I raised [this github issue](https://github.com/tarleb/parse-latex/issues/6) which describes this problem. 

Essentially, I have this `=latex` chunk in my document:

````
```{=latex}
\begin{center}
\begin{tabular}{|l|l|l|l|}
  \hline
  % after \\: \hline or \cline{col1-col2} \cline{col3-col4} ...
  Criterion & Formula &  Partial $\eta^2$   \\
  \hline
  Wilks's $\Lambda$ & $\Lambda = \prod^s_i \frac{1}{1+\lambda_i}$ &  $\eta^2 = 1-\Lambda^{1/s}$   \\
  Pillai trace & $V = \sum^s_i \frac{\lambda_i}{1+\lambda_i}$ &  $\eta^2 = \frac{V}{s} $   \\
  Hotelling-Lawley trace & $H = \sum^s_i \lambda_i$ & $\eta^2 = \frac{H}{H+s}$   \\
  Roy maximum root & $R = \lambda_1$  &  $ \eta^2 = \frac{\lambda_1}{1+\lambda_1}$  \\
  \hline
\end{tabular}
\end{center}
```
````

This is latex, so the normal `r` yaml options don't work, i.e., I can't use

```
#' label: tbl-stats
#' tbl-caption: Test statistics for multivariate tests
#' ...
```

I'm not sure how to do what I want as a `:::` div

**Comment by friendly** [comment]
Sorry, I meant

```
#| label: tbl-stats
#| tbl-caption: Test statistics for multivariate tests
#| ...
```

**Comment by mcanouil** [comment]
You probably need the cross-reference div syntax, see <https://quarto.org/docs/authoring/cross-references-divs.html>.
